documentation/src/main/asciidoc/topics/marshalling.adoc

```diff
@@ -123,6 +97,68 @@ JBoss Marshalling is deprecated and planned for removal in a future version of {
 interface when persisting data unless you configure JBoss marshalling. However, this interface is also deprecated and planned for removal.
 ====
 
+.Reference
+
+link:#deserialization_whitelist[Adding Java Classes to Deserialization White Lists]
+
+=== Kryo Marshalling
+{brandname} provides a marshalling implementation that uses Kryo libraries.
+
+Register custom Kryo schemas for object marshalling as follows:
+
+. Implement a service provider for the `SerializerRegistryService.java` interface.
+. Place all serializer registrations in the `register(Kryo)` method; where serializers are registered with the supplied `Kryo` object using the Kryo API, for example:
++
+----
+kryo.register(ExampleObject.class, new ExampleObjectSerializer())
+----
++
+. Specify the full path of implementing classes in your deployment JAR file within:
++
+----
+META-INF/services/org/infinispan/marshaller/kryo/SerializerRegistryService
+----
+
+//Community only
+ifndef::productized[]
+{brandname} provides a link:http://central.maven.org/maven2/org/infinispan/infinispan-marshaller-kryo-bundle/{infinispanversion}/infinispan-marshaller-kryo-bundle-{infinispanversion}.jar[Kryo Bundle] that includes all runtime class files for the Kryo marshalling implementation. Download the JAR and add to the `server/lib` directory for {brandname} servers.
```

Comment:
Nitpick, but should it be "and it to the `server/lib` directory of the {brandname} servers"?

This also applies to the Protostuff and storing Pojos section.